README.md

```diff
@@ -122,19 +114,20 @@ Check out the [full documentation site](https://docs.openzeppelin.com/contracts-
 - [Contract extensibility pattern](https://docs.openzeppelin.com/contracts-cairo/0.6.1/extensibility)
 - [Proxies and upgrades](https://docs.openzeppelin.com/contracts-cairo/0.6.1/proxies)
 - [Security](https://docs.openzeppelin.com/contracts-cairo/0.6.1/security)
-- [Utilities](https://docs.openzeppelin.com/contracts-cairo/0.6.1/utilities)
+- [Utilities](https://docs.openzeppelin.com/contracts-cairo/0.6.1/utilities) -->
 
 ### Cairo
 
-- [StarkNet official documentation](https://www.cairo-lang.org/docs/hello_starknet/index.html#hello-starknet)
-- [Cairo language documentation](https://www.cairo-lang.org/docs/hello_cairo/index.html#hello-cairo)
-- Perama's [Cairo by example](https://perama-v.github.io/cairo/by-example/)
-- [Cairo 101 workshops](https://www.youtube.com/playlist?list=PLcIyXLwiPilV5RBZj43AX1FY4FJMWHFTY)
+- [Cairo book](https://book.cairo-lang.org/)
+- [Cairo language documentation](https://docs.cairo-lang.org/)
+- [Starknet book](https://book.starknet.io/)
+- [Starknet documentation](https://www.cairo-lang.org/docs/hello_starknet/index.html#hello-starknet)
```

Comment:
```suggestion
- [Starknet documentation](https://docs.starknet.io/documentation/)
```

Broken link.

There's not much Starknet documentation in cairo-lang AFAICT. I think this is a better resource